### PR TITLE
Fix `Add-tag-to-commits` and `mark-merge-commits-in-list`

### LINKED ISSUE #3294: Marks merge commits in commit lists not working
Since the GitHub redesign, the _Marks merge commits in commit lists_ is not working anymore. 😭 
example from README

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #3294

This fixes both features that relied on the same dataset and the dom changed.

Test On:
https://github.com/sindresorhus/refined-github/commits/master

### CODE REVIEW

File: source/features/mark-merge-commits-in-list.tsx [hunk 1]
```diff
@@ -33,8 +33,11 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 	return mergeCommits;
 };
 
-function getCommitHash(commit: HTMLElement): string {
-	return commit.dataset.channel!.split(':')[3];
+// eslint-disable-next-line import/prefer-default-export
+export function getCommitHash(commit: HTMLElement): string {
+	// Pre "Repository refresh" layout
+	return 	commit.dataset.channel!.split(':')[3] ||
```
Comment (fregante) [inline source/features/mark-merge-commits-in-list.tsx:39]: I think this should be `??`

Comment (yakov116) [reply source/features/mark-merge-commits-in-list.tsx:39]: Channel still exists

Comment (yakov116) [reply source/features/mark-merge-commits-in-list.tsx:39]: Your right. It returns undefined - I responded half in my sleep.
```suggestion
	return 	commit.dataset.channel!.split(':')[3] ??
```

**Comment by fregante** [comment]
Thanks Yakow ^_^

**Pull request merged**